schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-1

```diff
@@ -0,0 +1,225 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-1",
+    "format" : "jsonschema"
+  },
+  "description": "The configuration file for the Snowplow mobile trackers.",
+  "type": "object",
+  "properties": {
+    "configurationVersion": {
+      "description": "Version number that identifies the current configuration. It has to be increased on each update.",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
```

Comment:
We can probably change this in the future. It would be good if we transform this in a string in the future so it will only check if the string is different.